staging/src/k8s.io/apiserver/pkg/audit/request.go

```diff
@@ -171,7 +171,12 @@ func LogResponseObject(ae *auditinternal.Event, obj runtime.Object, gv schema.Gr
 		return
 	}
 	if status, ok := obj.(*metav1.Status); ok {
-		ae.ResponseStatus = status
+		ae.ResponseStatus = &metav1.Status{
+			// Selectively copy the bounded fields to avoid duplicate status in audit events
```

Comment:
what do we lose in practice?

Reply:
Before this change:
``` json
    "responseStatus": {
        "apiVersion": "v1",
        "code": 403,
        "details": {
            "kind": "pods"
        },
        "kind": "Status",
        "message": "pods is forbidden: User \"tom\" cannot list pods in the namespace \"default\"",
        "metadata": {},
        "reason": "Forbidden",
        "status": "Failure"
    },
```

After this change:
```json
    "responseStatus": {
        "code": 403,
        "metadata": {},
        "reason": "Forbidden",
        "status": "Failure"
    },
```

Reply:
Why do you want to lose the message?

Reply:
As @tallclair said:
https://github.com/kubernetes/kubernetes/issues/60108#issue-298796763

Reply:
And is this message duplicated as well? If yes, I can agree with the change.

Reply:
I also think Message is necessay.
Add it back.